modules/nf-core/beagle/beagle5/main.nf

```diff
@@ -0,0 +1,55 @@
+process BEAGLE_BEAGLE5 {
```

Comment:
I actually had just BEAGLE to begin with, but then realised that Beagle has another tool [bref3.22Jul22.46e.jar](http://faculty.washington.edu/browning/beagle/bref3.22Jul22.46e.jar) [here](http://faculty.washington.edu/browning/beagle/beagle.html). So when someone makes another module for the "bref3" tool, they can put it inside Beagle module folder. In addition Beagle has two main releases/versions, the old v4 and the newer v5, so I changed it to BEAGLE_BEAGLE5 to reflect these things. But it's also totally fine to just have one Beagle and then another separate module (NOT as tool/subtool) for bref3... What do you think?

Reply:
I think just `BEAGLE_BEAGLE` is fine then :) no need to specify the version there IMO since we mostly just work with the most recent version (except if there is a good reason for both versions to be separate modules?)